pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -1433,6 +1434,29 @@ let
         };
       };
 
+      jebbs.plantuml = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "plantuml";
+          publisher = "jebbs";
+          version = "2.17.4";
+          sha256 = "sha256-fnz6ubB73i7rJcv+paYyNV1r4cReuyFPjgPM0HO40ug=";
+        };
+        nativeBuildInputs = [ jq moreutils ];
+        postInstall = ''
+          cd "$out/$installPrefix"
+          jq '.contributes.configuration.properties."plantuml.java".default = "${plantuml}/bin/plantuml"' package.json | sponge package.json
+        '';
+
+        meta = with lib; {
+          description = "Rich PlantUML support for Visual Studio Code";
```

```suggestion
          description = "A Visual Studio Code extension to support Rich PlantUML";
```
* Please correct my sentence if it is wrong. Goal here was to say it is a VSCode extension.

---

On this point, I just used the extension's description.

---

Understandable.

---

Realize nixpkgs is a general repository of packages. VSCode marketplace is dedicated to VSCode extensions. So it makes for nixpkgs to point out a package is a VSCode extension in description.